.env.example

```diff
@@ -1,3 +1,5 @@
 VITE_CKAN_URL=https://data.stadt-zuerich.ch
 VITE_ANALYTICS_SRC=''
 VITE_ROBOTS_DIRECTIVE=noindex
+VITE_BACKEND_URL = https://ckan-ogdzh.clients.liip.ch
```

Comment:
We don't need a new env variable for this - this is what `VITE_CKAN_URL` is for. (The example and fallback values `VITE_CKAN_URL` tend to point at the frontend url, but this is because Sascha set them up before we had the separate frontend and backend urls.)